streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -208,6 +214,29 @@ public synchronized void close() {
     }
 
     /**
+     * Cleans up local state store directory ({@code state.dir}), by deleting all data with regard to the application-id.
+     * <p>
+     * May only be called either before instance is started or after instance is closed.
+     *
+     * @throws IllegalStateException if instance is currently running
+     */
+    public void localCleanUp() {
```

Comment:
Sure. I wanted to have a more clarifying name. But no strong opinion in general... Will change it.